test/libyul/yulInterpreterTests/dataoffset.yul

```diff
@@ -10,5 +10,5 @@ object "main"
 // Trace:
 // Memory dump:
 // Storage dump:
-//   0000000000000000000000000000000000000000000000000000000000000000: 000000000000000000000000000000000000000000000000000000000000006e
```

Comment:
It seems this is not properly fixed yet since we need to pass the values to the builtin evaluator.